services/web/server/tests/unit/with_dbs/01/test_catalog_api.py

```diff
@@ -3,58 +3,100 @@
 # pylint:disable=redefined-outer-name
 
 import re
-from typing import Callable
+from typing import Type
 
 import pytest
 from aiohttp import web
+from aiohttp.test_utils import TestClient
 from pytest_simcore.helpers.utils_assert import assert_status
-from simcore_service_webserver.application import (
-    create_safe_application,
-    setup_catalog,
-    setup_db,
-    setup_login,
-    setup_products,
-    setup_rest,
-    setup_security,
-    setup_session,
-)
-from simcore_service_webserver.application_settings import setup_settings
+from pytest_simcore.helpers.utils_login import UserInfoDict
 from simcore_service_webserver.catalog_settings import (
     CatalogSettings,
     get_plugin_settings,
 )
 from simcore_service_webserver.db_models import UserRole
 
-
-@pytest.fixture()
-def client(
-    event_loop,
-    app_cfg,
-    aiohttp_client,
-    postgres_db,
-    monkeypatch_setenv_from_app_config: Callable,
-):
-    # fixture: minimal client with catalog-subsystem enabled and
-    #   only pertinent modules
-    #
-    # - Mocks calls to actual API
-
-    monkeypatch_setenv_from_app_config(app_cfg)
-    app = create_safe_application(app_cfg)
-    assert setup_settings(app)
-
-    # patch all
-    setup_db(app)
-    setup_session(app)
-    setup_security(app)
-    setup_rest(app)
-    setup_login(app)  # needed for login_utils fixtures
-    assert setup_catalog(app)
-    setup_products(app)
-
-    yield event_loop.run_until_complete(
-        aiohttp_client(app, server_kwargs={"port": app_cfg["main"]["port"]})
-    )
+# @pytest.fixture
+# def app_cfg(default_app_cfg: ConfigDict, unused_tcp_port_factory: Callable):
```

Comment:
Forgot to delete? Why commented

Reply:
ah yes, that is what I was trying to ask you this morning...

Reply:
I would like to discuss app_cfg and client fixtures, cause I find them a bit confusing. and I wonder if I'm not missing something there.